README.md

```diff
@@ -553,13 +553,13 @@ client.host('mio.nexmo.com') #Change host to mio.nexmo.com - this change will be
 These attributes are private in the client class and the only way to access them is using the getters/setters we provide.
 
 ```python
-from nexmo import Client
+from vonage import Client
 
 client = Client(key='YOUR_API_KEY', secret='YOUR_API_SECRET')
-print(client.host()) # return rest.nexmo.com
-client.host('mio.nexmo.com') # rewrites the host value to mio.nexmo.com
-print(client.api_host()) # returns api.nexmo.com
-client.api_host('myapi.nexmo.com') # rewrite the value of api_host
+print(client.host()) # return rest.vonage.com
+client.host('mio.vonage.com') # rewrites the host value to mio.vonage.com
+print(client.api_host()) # returns api.vonage.com
+client.api_host('myapi.vonage.com') # rewrite the value of api_host
```

Comment:
Should swap this with an obvious testing URL instead of what looks like a legit one using our domain